packages/components/src/components/segmented-control/segmented-control/segmented-control.tsx

```diff
@@ -33,7 +34,7 @@ export class SegmentedControl {
   @Element() public host!: HTMLElement;
 
   /** Background color variations */
-  @Prop() public backgroundColor?: SegmentedControlBackgroundColor = 'background-default';
+  @Prop() public backgroundColor?: SegmentedControlBackgroundColor = 'background-default'; // 'background-color' prop is deprecated
```

Comment:
Having no default anymore would probably simplify things a lot.

Reply:
fixed it